CONTRIBUTING.md

```diff
@@ -88,6 +92,34 @@ nvm use
 npm install
 ```
 
+#### Running the integration tests installation script
```

Comment:
Should we move this installation script instructions after [line 239](https://github.com/Parsely/wp-parsely/pull/1211/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R239) which already have steps about running integration tests?

Reply:
This was done on purpose to have all installation-related instructions in one place (section "Setting up your local development environment"), where we describe how to install everything.

Then we have the "Developing locally" section that actually describes how to use all the stuff we've installed in the previous section (linting, testing, etc) 😃

I think that by making every section focused, it makes the document easier to follow. But I'm open to feedback if you think otherwise.

Reply:
I don't have strong preference, completely fine with your decision.